python/dask_cudf/dask_cudf/core.py

```diff
@@ -753,14 +757,11 @@ def from_dask_dataframe(df):
     "rpow",
 ]:
```

Comment:
```suggestion
):
```

Same here, can we switch to `tuple`? I can't code-suggest line 743 for `(`.